### ISSUE TITLE
Test in the State wrapper

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/my-solar-system#289]
We have not yet tested My Solar System sufficiently in the State wrapper.  @AgustinVallejo and I shoud each (independently) spend ~30 minutes on this.   This should be completed before asking @arouinfar to review, since resolving issues can dramatically affect the PhET-iO instrumentation.

@AgustinVallejo I'll give you a demo of how to test, what to look for, etc. in our next pairing session.

**Comment by pixelzoom** [comment]
@AgustinVallejo and I took a tour of the State wrapper in a brief pairing session this morning.

**Comment by AgustinVallejo** [comment]
I've been testing with the state wrapper for a while and never found anything to report on. Unassigning myself.

**Comment by pixelzoom** [comment]
I did a thorough test with the State wrapper.  The only thing that I found that was not behaving correctly is the KeypadDialog. See screenshot below, where the range and value are both missing.  But that is to be expected, since we punted on instrumenting KeypadDialog and Keypad (see #234).

<img width="520" alt="screenshot_2893" src="https://github.com/phetsims/my-solar-system/assets/3046552/86f40cea-5d73-4016-9783-52e694491655">
